### DISCUSSION TITLE
How to use ground pi to control air Pi?

### DISCUSSION THREAD

**Discussion by return97** [svpcom/wfb-ng#89] [Questions]
I want to use SSH to connect ground pi to control air Pi?

How to do it?

I tried to use UDP socket but couldn't connect?

Do I need to add some external settings to control the air Pi?

thank you!

**Comment by svpcom** [comment]
You need to use IPoverWB tunnel. See https://github.com/svpcom/wifibroadcast/wiki/Setup-HOWTO [10]
